src/modules/common/containers/Helmet.js

```diff
@@ -11,14 +11,16 @@ import ReactHelmet from 'react-helmet'
 import type { Location } from 'redux-first-router'
 import getLanguageChangePath from '../../app/getLanguageChangePath'
 
-type PropsType = {
+type PropsType = {|
   title: string,
   categories: CategoriesMapModel,
   events: Array<EventModel>,
   languages: Array<LanguageModel>,
   location: Location,
-  metaDescription?: string
-}
+  metaDescription?: string,
+  // Custom redux state for testing purposes
+  store?: StateType
```

Comment:
This really shouldn't be here. Can we find a way to avoid this?

Reply:
Done